src/main/resources/swagger.api/mod-fqm-manager.yaml

```diff
@@ -44,6 +44,19 @@ paths:
           $ref: '#/components/responses/badRequestResponse'
         '500':
           $ref: '#/components/responses/internalServerErrorResponse'
+  /materialized-views/refresh:
+    post:
+      operationId: refreshMaterializedViews
+      tags:
+        - materializedViews
+      description: Refresh all materialized views for a tenant.
+      responses:
+        '204':
+          description: 'Views refreshed'
+        '400':
+          $ref: '#/components/responses/badRequestResponse'
+        '500':
+          $ref: '#/components/responses/internalServerErrorResponse'
```

Comment:
Thanks for putting this here :) I meant to say something about it definitely not going in folio-query-tool-metadata (since that would put it in the edge module, too), and totally forgot